### ISSUE TITLE
Specto & Library Updates

### ISSUE THREAD

**Issue by waynemcke** [mrknow/filmkodi#478]
I use a TV Shows Library that was previously set up with Genesis and has now been set up with Specto for the last 6 months. After the switch to Specto, everything worked perfectly but over the last couple of months, I've noticed a problem that seems to be getting worse.

The issue is with my TV Shows Library not being consistent when it updates new episodes from Specto. Some shows receive the library update for a new episode very quickly whereas others may not update in the library for days/weeks after they appear on Specto and sometimes not at all. 

An example is Cold Feet (UK TV Show - S06E01) which I can find in Specto and was aired on 5th Sept. My library updates every 4 hours but 4 days later and after maybe 24 automated updates, it is still not showing in my Library. If I run a manual update, it will also not add it to my library.

The only way I can fix this is to manually delete the whole show out of my library, delete the files off my hard drive and then add the show back to the library again. Doing this will import every episode that is available on Specto including the previously missing episode. But I don't understand why I have to do this when other shows are importing automatically and correctly.

One clue that may help is that all the affected shows are UK shows.....but not all of the UK shows are affected. US shows tend to update within 24 hours on a consistent basis???

I've done a reinstall and clean library but it hasn't solved anything. Does anyone have any ideas of what else I can try or how to fix this as I don't know if it's Specto, TVDB or my Library that's at fault.

Thanks very much for your time.

**Comment by waynemcke** [comment]
....and to add something extra, I've just manually deleted 2 shows to add them both back to the library to capture the new episodes. One of them loaded back in without a problem and the new episode was included in the update. The other show now won't scan back in at all despite me deleting the show/files........it says 'adding to library' but when the scan starts, it pauses for a few seconds and the stops scanning? I've tried to rescan it a few times and the same result each time?

I'm wondering if I'm going to need to do a fresh install again but would prefer to try any other options first to save me some time......

Thanks again.
